packages/itwinui-css/backstop/tests/code.html

```diff
@@ -45,6 +45,7 @@ <h2>Codeblock</h2>
       >
         <div class="iui-title-bar">
           <span class="iui-title">HTML</span>
+
```

Comment:
Formatter made this one?

Reply:
Must have been a mistake on my part, it is removed now.